community/kernel/src/main/java/org/neo4j/kernel/api/cursor/PropertyCursor.java

```diff
@@ -75,14 +102,19 @@ public void close()
         }
     };
 
-
-    /**
-     * Move the cursor to a particular property.
-     *
-     * @param keyId of the property to find
-     * @return true if found
-     */
     boolean seek( int keyId );
```

Comment:
No, no reason really. Can't remember why I removed that, to be honest. At some point all of these methods need more Javadoc.